sopel/modules/meetbot.py

```diff
@@ -238,17 +273,24 @@ def endmeeting(bot, trigger):
     See [meetbot module usage]({% link _usage/meetbot-module.md %})
     """
     if not ismeetingrunning(trigger.sender):
-        bot.say('Can\'t do that, start meeting first')
+        bot.say("Can't do that, start meeting first")
         return
     if not ischair(trigger.nick, trigger.sender):
         bot.say('Only meeting head or chairs can do that')
         return
     meeting_length = time.time() - meetings_dict[trigger.sender]['start']
-    # TODO: Humanize time output
-    bot.say(bold("Meeting ended!") + " total meeting length %d seconds" % meeting_length)
+    bot.say(
+        bold("Meeting ended!") +
+        " Total meeting length %d minutes" % (meeting_length // 60))
     logHTML_end(trigger.sender)
-    htmllog_url = meeting_log_baseurl + quote(trigger.sender + '/' + figure_logfile_name(trigger.sender) + '.html')
-    logplain('Meeting ended by %s, total meeting length %d seconds' % (trigger.nick, meeting_length), trigger.sender)
+    htmllog_url = meeting_log_baseurl + quote(
+        trigger.sender + '/' + figure_logfile_name(trigger.sender) + '.html'
+    )
+    logplain(
+        'Meeting ended by %s, total meeting length %d minutes'
```

Comment:
I noticed several comma splices like this while scanning the diff. Bonus points if you fix those along with your other tweaks!